_posts/2018/2018-04-11-uklidmecesko.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+category: CLANKY
+title: Uklízíš? Zaplatíš! 
+date: 2018-04-10T10:10:00+01:00  
+tags: 
+    - Ukliďme Česko 
+    - Služby města Pardubic
+author: Piráti Pardubického kraje 
+image: assets/img/articles/2018/uklidmecesko2018pce.jpg #751x422
+
+---
+
+
+Piráti se v Pardubickém kraji připojili k akci Ukliďme Česko. 
+V některých městech, jako třeba v Pardubicích, Chrudimi nebo Svitavách, 
+přímo jako organizátoři. V jiných městech a vsích, jako třeba v 
+České Třebové, Moravské Třebové nebo v Sezemicích, se připojili k 
+akcím místních spolků a organizací. Celkem se zapojilo přes 50 dobrovolníků 
+a sesbíraly se metráky odpadu. A zatímco v některých městech nám odpad bez 
+problémů  odvezly místní technické služby, v Pardubicích nám odpad 
+odvézt odmítli.
+
+> "Zorganizovali jsme úklid v Bubeníkových sadech na trase až k lávce 
+pod nemocnicí po obou březích Chrudimky. Úklidu se zúčastnilo přibližně 
+25 dobrovolníků, z toho 8 dětí, převážně z řad lidí, které jsme do té 
+doby neviděli – a za jejich velké nasazení samozřejmě děkujeme. Abychom 
+nezatěžovali už tak napjatý rozpočet organizátorů, nakoupili jsme pro 
+účastníky vše potřebné ze svého. Vzhledem k přibývajícímu počtu nahlášených 
+dobrovolníků jsme přibližně tři týdny před akcí požádali 
+[Služby města Pardubic] o informace k podpoře, která je akci 
+[Ukliďme Česko] ze strany města poskytována. Předpokládali 
+jsme vstřícný a aktivní přístup," 
+
+říká [Ondřej Karas], člen Pirátské strany, organizátor 
+Pardubického úklidu a aspirant na lídra komunální kandidátky 
+v tomto statutárním městě.
+
+Odpověď však tak vstřícná nebyla: 
+
+> "Dobrý den, pane Karasi, minulý týden jsem Vám slíbila vyjádření 
+města ohledně možné podpory úklidu města v rámci akce [Ukliďme Česko]. 
+Po zaslaném stanovisku města Vám proto musím napsat, že město bude 
+hradit akce, které organizují nezávislé organizace, zájmové spolky, 
+volná sdružení občanů apod... akce úklidu organizovaného Českou
+pirátskou stranou nebude městem z tohoto důvodu podpořena."
+
+Kontejner, který k velké radosti využily i jiné uklízející party, 
+tedy Piráti objednali a zaplatili ze svého. Stejně jako potřebné 
+pytle a ochranné pomůcky.
+
+> "Silně se ohrazuji proti postupu SmP-Odpady a.s. Nevíme, jakou 
+má spojitost akce Ukliďme Česko s politickou orientací svých organizátorů, 
+a nechápeme, jaký je rozdíl mezi úklidy, které jsou pořádány starosty obvodů, 
+a těmi, za kterými stojí politické strany nebo církve. Tato společnost si 
+zřejmě plně neuvědomuje, že naši dobrovolníci dělají práci, kterou by měli 
+odvádět oni, a drze po nás chce peníze za odvoz nasbíraného odpadu."
```

Comment:
Je nutne se takhle navazet do odpadu? Kdyz uz se k tomu vyjadril Ondra i odpady... Spis by bylo dobre, kdyby se Pavel vyjadril k tomu, jak to probihalo v Chrudimi.